internal/cli/apis.go

```diff
@@ -152,9 +154,10 @@ func showApiCmd(cli *cli) *cobra.Command {
 		Use:   "show",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Show an API",
-		Long:  "Show an API.",
-		Example: `auth0 apis show 
-auth0 apis show <id|audience>`,
+		Long:  "Display the name, scopes, token lifetime, and other information about an API.",
```

Comment:
Some long descriptions start with the short one, others don't. Consider sticking to a single style in a future PR.